oath/README.md

```diff
@@ -54,8 +55,13 @@ And use it to create `OathApplication` to select OATH applet on YubiKey
     
 ```
 
-The `OathApplication` provides a method for every command from the [YK OATH protocol](https://developers.yubico.com/OATH/YKOATH_Protocol.html) to add, remove, list and calculate credentials. In addition to these requests, the `OathApplication` provides an interface for setting/validating a password on credential storage, calculating all credentials and resetting the OATH application to its default state. For the complete list of methods look at the `OathApplication` code level documentation.  
-**OATH YubiKit** also provides a class for defining an OATH `Credential`, which has a convenience initializer `Credential.parseUri` which can parse the credential parameters from Uri of [Key Uri Format](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
+Next, use the `OathApplication` to add, remove, list, and calculate credentials.
+
+The `OathApplication` implements the YKOATH protocol. Refer to the [YKOATH protocol specification](https://developers.yubico.com/OATH/YKOATH_Protocol.html) for more details. 
+
+In addition, the `OathApplication` provides an interface for setting/validating a password on credential storage, calculating all credentials and resetting the OATH application to its default state. For the complete list of methods look at the [`OathApplication` class documentation](src/main/java/com/yubico/yubikit/oath/OathApplication.java).  
+
+The **OATH** module also provides a class for defining an OATH `Credential`, which has a convenience initializer `Credential.parseUri` which can parse the credential parameters from Uri of [Key Uri Format](https://github.com/google/google-authenticator/wiki/Key-Uri-Format).
```

Comment:
May be rephrase it somehow to avoid double which?

Reply:
fixed in f138d2e